README.md

```diff
@@ -255,12 +255,11 @@ partial assembly option (`-pa`).
 
 Some sample runs in 2D and 3D respectively are:
 ```sh
-mpirun -np 8 laghos -p 1 -m data/square01_quad.mesh -rs 3 -tf 0.8 -no-vis -pa
-mpirun -np 8 laghos -p 1 -m data/cube01_hex.mesh -rs 2 -tf 0.6 -no-vis -pa
+mpirun -np 8 laghos -p 1 -m data/square01_quad.mesh -rs 3 -tf 0.8 -pa
+mpirun -np 8 laghos -p 1 -m data/cube01_hex.mesh -rs 2 -tf 0.6 -vis -pa
 ```
 
-The latter produces the following density plot (when run with the `-vis` instead
-of the `-no-vis` option)
+The latter produces the following density plot (notice the `-vis` option)
```

Comment:
I like the Gresho vortex problem! Can we add a section on it, maybe with one of the following images?

![glvis_s01](https://user-images.githubusercontent.com/1278247/46233509-bb668880-c327-11e8-8ca3-a78f483d7d02.png)
![glvis_s03](https://user-images.githubusercontent.com/1278247/46233507-bb668880-c327-11e8-8172-f85ddecbe8f3.png)
![glvis_s02](https://user-images.githubusercontent.com/1278247/46233508-bb668880-c327-11e8-9d44-bfabb0d95bc5.png)